bot/cogs/snakes.py

```diff
@@ -11,8 +13,22 @@ class Snakes:
     """
     Snake-related commands
     """
+    python_info = '''
+                    Python (Programming Language)
```

Comment:
You should probably remove all these huge indents.

Reply:
You also don't need the `\n`s in here. Those are taken from how you've formatted the string.